load.py

```diff
@@ -0,0 +1,26 @@
+from audioop import mul
```

Comment:
This is not standard practice so I cant blame yall for not doing it, but I would love some documentation at top explaining the class purpose, even if it is not super complicated